103_lineshape_broadening.ipynb

```diff
@@ -0,0 +1,874 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from radis.lbl.broadening import gaussian_lineshape\n",
```

Comment:
We could mention the function in the Markdown, with a link to the radis documentation ? https://radis.readthedocs.io/en/latest/source/radis.lbl.broadening.html#radis.lbl.broadening.gaussian_lineshape